src/CMakeLists.txt

```diff
@@ -41,6 +41,12 @@ if (${Kokkos_ENABLE_CUDA})
   set(PAR_LOOP_LAYOUT_VALUES "MANUAL1D_LOOP;MDRANGE_LOOP;TPTTR_LOOP;TPTTRTVR_LOOP"
     CACHE STRING "Possible loop layout options.")
 
+  set(PAR_INNER_LOOP_LAYOUT "TTR_INNER_LOOP" CACHE STRING
+    "Default loop layout for par_inner_for wrapper")
+
+  set(PAR_INNER_LOOP_LAYOUT_VALUES "TTR_INNER_LOOP"
+    CACHE STRING "Possible inner loop layout options.")
+
```

Comment:
Perhaps we should provide some documentation in the help file explaining why TTR and simdfor are the only options? I can see a new user getting confused by this.

Reply:
I realize after looking into it, a `TeamVectorRange`, which squashes `TeamThreadRange` and `ThreadVectorRange`, is more correct here. I'm making that change.